bochs/.bochsrc

```diff
@@ -1271,10 +1271,23 @@ speaker: enabled=1, mode=sound, volume=15
 #=======================================================================
 # MAGIC_BREAK:
 # This enables the "magic breakpoint" feature when using the debugger.
-# The useless cpu instruction XCHG BX, BX causes Bochs to enter the
+# The useless cpu instructions XCHGW %REGW, %REGW causes Bochs to enter the
 # debugger mode. This might be useful for software development.
 #
-# Example:
+# Register number id:
+#   0: DISABLE MAGIC_BREAK
```

Comment:
Hi,

I haven't even notice these pull requests.
For some reason I didn't receive a notification from github,
This one is fine (with small adjustments to make to match general Bochs coding style).
How about adding capability to control/toggle them from inside the debugger interface ?
How about enabling more than one together ?
I think it should be a bitmask of 8 bits instead of bool in bx_dbg.magic_break and checking smth like
((1<<i->dst()) & bx_dbg.magic_break)

Reply:
awesome idea! I will do it :-)